pocketpy/PSPBUILD

```diff
@@ -1,24 +1,25 @@
 pkgname=pocketpy
-pkgver=1.0.9
-pkgrel=1
+pkgver=1.4.5
+pkgrel=2
```

Comment:
Since the version changed, the pkgrel should be 1.

Reply:
Done. I forgot about that, thanks!